core/templates/dev/head/pages/error/error.mainpage.html

```diff
@@ -60,7 +60,8 @@ <h2 ng-if="statusCode === 500" class="oppia-error-h2" translate="I18N_ERROR_MESS
 
 {% block footer_js %}
   {{ super() }}
-  <script src="/templates/dev/head/pages/error/Error.js"></script>
+  <script src="/templates/dev/head/pages/error/error.module.js"></script>
```

Comment:
Are the scripts required here?

Reply:
I do not think they are needed because it is already there in base.html but based on the travis tests I think it is needed because if the file loaded 2 times then I should have got errors because alerts and similar actions would occur 2 times instead of once. 
But I am not sure about this, I will tru removing it and checking. If it works Ill remove it.

Reply:
done